### PR TITLE
Added test for updated styles on mobile contribution type tabs

### PR DESCRIPTION
**Pull request by jlieb10**
## Why are you doing this?
Using a test to update to the styles for tabs on mobile devices for 50% of users (vertical instead of horizontal below 440px) - Trello card attached

[**Trello Card**](https://trello.com/c/PVwwQgzt/332-implement-on-mobile-stack-frequency-types)

## Changes

* Stacking the contribution tabs below the 440px breakpoint

## Screenshots
Control:
verticalContributionsTest_control

Variant:
verticalContributionsTest_variant

### CODE REVIEW

**Comment by jlieb10** [comment]
@ionamckendrick @jranks123 I refactored this to be a test. Screenshots of control and variant above.

File: support-frontend/assets/helpers/abTests/abtestDefinitions.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ import type { Tests } from './abtest';
 
 
 export type EditorialiseAmountsVariant = 'control' | 'averageAnnualAmount' | 'monthlyBreakdownAnnual' | 'weeklyBreakdownAnnual' | 'notintest';
+export type VerticalContributionsTabsVariant = 'control' | 'verticalTabsMobile';
```
Comment (jranks123) [inline support-frontend/assets/helpers/abTests/abtestDefinitions.js]: you need to add `notintest`

File: support-frontend/assets/pages/new-contributions-landing/contributionsLanding.scss [hunk 2]
```diff
@@ -1220,3 +1203,30 @@ form {
 .editorialise-amounts-test-copy:empty {
   display: none;
 }
+
+// Vertical contributions tabs test
+.form__radio-group--mobile-vertical-tabs .form__radio-group-list {
+  flex-flow: column-reverse nowrap;
```
Comment (joelochlann) [inline support-frontend/assets/pages/new-contributions-landing/contributionsLanding.scss]: with sass you can put all of this inside the parent

```sass
.form__radio-group--mobile-vertical-tabs {
   .form__radio-group-list {

  }

  .form__radio-group-list--border {

  }
}
```

Comment (jlieb10) [reply support-frontend/assets/pages/new-contributions-landing/contributionsLanding.scss]: @joelochlann that's how I would normally have written it but I thought the Guardian doesn't like nested styles-- there's no other nested styles in the sheet (I think) except for media queries

Comment (jlieb10) [reply support-frontend/assets/pages/new-contributions-landing/contributionsLanding.scss]: @joelochlann nevermind - i will refactor

**Pull request merged**